modules/qt/hm.nix

```diff
@@ -0,0 +1,97 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+{
+  options.stylix.targets.qt = {
+    enable = config.lib.stylix.mkEnableTarget "QT" pkgs.stdenv.hostPlatform.isLinux;
+    platform = lib.mkOption {
+      description = ''
+        Platform for QT.
+
+        Defaults to the standard platform of the configured DE in NixOS when
+        `stylix.homeManagerIntegration.followSystem = true`.
+
+        Fallback to qtct.
+      '';
+      type = lib.types.str;
+      default = "qtct";
+    };
+  };
+
+  config = lib.mkIf config.stylix.targets.qt.enable (
+    let
+      cfg = config.stylix;
+      iconTheme =
+        if (cfg.polarity == "dark") then cfg.iconTheme.dark else cfg.iconTheme.light;
+
+      recommendedStyle = {
+        gnome = if config.stylix.polarity == "dark" then "adwaita-dark" else "adwaita";
+        kde = "breeze";
+        qtct = "kvantum";
+      };
```

Comment:
I assume this is what we plan to support for now, right? Would it make sense for end-users to be able to set any value?

Reply:
No these would be all the values that would have a defined behavior. It shouldn't break from arbitrary values though. It would just do nothing instead.

Reply:
> No these would be all the values that would have a defined behavior. It shouldn't break from arbitrary values though.

OK.

> It would just do nothing instead.

Should we leverage `assertions` or `warnings` in these cases or silently ignore it?

Reply:
Why not. I'll add it.

Reply:
Actually, it seems like NixOS already has a sanity check for that. HM doe not though